### PR TITLE
[RFR] Color preview

### PR DESCRIPTION
**Pull request by HugoGiraudel**

### CODE REVIEW

**Comment by HugoGiraudel** [comment]
@ben-eb @SassDoc/owners Please review. :)

**Comment by ben-eb** [comment]
LGTM. :+1:

**Comment by HugoGiraudel** [comment]
#42 by the way.

File: package.json [hunk 1]
```diff
@@ -20,11 +20,13 @@
...
   ],
   "dependencies": {
+    "chroma-js": "^0.6.3",
     "extend": "1.*",
     "sassdoc-extras": "2.0.0-rc.4",
+    "themeleon": "^3.0.0",
     "swig": "1.4.0",
     "swig-extras": "0.*",
-    "themeleon": "^3.0.0"
+    "themeleon": "3.*"
```
Comment (ben-eb) [inline package.json]: `themeleon` is listed twice. :smile:

**Comment by valeriangalliat** [comment]
Awesome!

**Pull request merged**